vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -105,15 +113,50 @@ public void test_deleteImageBlockById_forNonExistentId() throws BlockDoesNotExis
         Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
         when(contentBlockRepository.findById("notARealId")).thenReturn(contentBlockOptional);
         Mockito.doThrow(EmptyResultDataAccessException.class).when(imageBlockRepo).deleteById(imageBlockId);
-        managerToTest.deleteImageBlockById(imageBlockId,"slideId_1");
+        managerToTest.deleteImageBlockById(imageBlockId, "slideId_1");
     }
 
     @Test
     public void test_deleteImagetBlockById_whenIdIsNull() throws BlockDoesNotExistException {
         String imageBlockId = null;
-        managerToTest.deleteImageBlockById(null,"slideId_1");
+        managerToTest.deleteImageBlockById(null, "slideId_1");
         Mockito.verify(imageBlockRepo, Mockito.never()).deleteById(imageBlockId);
 
     }
 
+    @Test
+    public void test_adjustContentOrder_success() throws BlockDoesNotExistException {
+        ContentBlock contentBlock_1 = new ContentBlock();
+        contentBlock_1.setId("contentBlockId_1");
+        contentBlock_1.setContentOrder(Integer.valueOf(3));
+        ContentBlock contentBlock_2 = new ContentBlock();
+        contentBlock_2.setId("contentBlockId_2");
+        contentBlock_2.setContentOrder(Integer.valueOf(4));
+        List<ContentBlock> contentBlocks = new ArrayList<>();
+        contentBlocks.add(contentBlock_1);
+        contentBlocks.add(contentBlock_2);
+        Optional<ContentBlock> contentBlockOptional_1 = Optional.of(contentBlock);
+        Optional<ContentBlock> contentBlockOptional_2 = Optional.of(contentBlock_1);
+        when(contentBlockRepository.findById("contentBlockId_1")).thenReturn(contentBlockOptional_1);
+        when(contentBlockRepository.findById("contentBlockId_2")).thenReturn(contentBlockOptional_2);
```

Comment:
merge 140-141. with 138-139, then use the actual objects (not the optionals) further down.